src/components/Popover/index.test.js

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, waitFor, screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 
 import Popover from '.';
 
 describe('Popover component', () => {
-  it('tests that trigger text gets rendered', () => {
+  it('tests that trigger text gets rendered', async () => {
     render(
       <Popover triggerElement={<button>trigger</button>}>
         <p>popover content</p>
       </Popover>
     );
     expect(screen.queryByText('popover content')).toBeNull();
-    expect(screen.getByText('trigger')).toBeTruthy();
+    await waitFor(() => expect(screen.getByText('trigger')).toBeTruthy());
   });
 
-  it('tests that trigger and content text gets rendered', () => {
+  it('tests that trigger and content text gets rendered', async () => {
     render(
       <Popover isOpen triggerElement={<button>trigger</button>}>
         <p>popover content</p>
       </Popover>
     );
-    expect(screen.getByText('popover content')).toBeTruthy();
     expect(screen.getByText('trigger')).toBeTruthy();
+    await waitFor(() => expect(screen.getByText('popover content')).toBeTruthy());
```

Comment:
`async`, `await`, and `waitFor` ALL must be used here in order to maintain the integrity of the test now that `LazyTippy` is being used. If they are not, this will trigger a bunch of `act` errors and "popover content" also won't be present in the DOM tree.